cli/Makefile

```diff
@@ -39,7 +40,7 @@ endif
 	gox -ldflags '-X github.com/hasura/graphql-engine/cli/version.BuildVersion=$(VERSION) -X github.com/hasura/graphql-engine/cli/plugins.IndexBranchRef=$(PLUGINS_BRANCH) -s -w -extldflags "-static"' \
 	-rebuild \
 	-os="$(OS)" \
-	-arch="amd64" \
+	-arch="$(ARCH)" \
```

Comment:
Probably using `-osarch` is a better option. I doubt `OS` is currently being used but dropping it could be breaking